locales/en.default.schema.json

```diff
@@ -2825,7 +2825,8 @@
         }
       },
       "presets": {
-        "name": "Rich text"
+        "name": "Rich text",
+        "name-heading": "Heading"
```

Comment:
I couldn't find another example of two section presets like this, so I wasn't sure what our standard naming practice is for these entries. I went with `name-heading` for now, but I'm looking for other recommendations!